content/featured/2023-08-12-rebrand.md

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
for now i think we can stick to having 2 copies (one under `featured` and the other in `post`) for featured posts, in the future lets refactor so we don't have to maintain 2 copies

> alternative: do a relative path symlink or something?

Reply:
yup agree, it's very clunky Ron